CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Changelog
 =============
 
-## 2016/05/11 - Version 0.1
+## 2016/05/12 - Version 0.1
```

Comment:
AFAIK the change log should contains info until the latest tag released (v0.1 in my case), I just want to start adding functionalities under a new snapshot version.

Anyway, yesterday @fedefernandez and I were discussing about the proper protocol to do that, in order to enrich the document I'm writing for **How to publish open source libraries, step by step**.